src/scss/lr7-classic-theme/_icons.scss

```diff
@@ -1,5 +1,5 @@
 .icon-monospaced {
-	display: inline-block;
+	display: inline-block !important;
```

Comment:
You should only use !important as a last resort. I think its better not to include it here.